README.md

```diff
@@ -149,12 +145,12 @@ The installation prerequisites are the same as described in the TensorFlow
         git checkout v0.6.1
```

Comment:
This is more of a comment from @sayantan-nervana , we were discussing this yesterday. Is their a constraint, to clone ngraph-tf in the *parent* directory of tensorflow repo, as mentioned in Step 6? If not, we can remove this part "one level above -- in the  *parent* directory of the `tensorflow` repo cloned in step 1:"

We might want to keep the ``` cd ..``` in the instructions to exit out of the tensorflow repo.